rundeckapp/src/test/groovy/rundeck/services/ProjectServiceSpec.groovy

```diff
@@ -2297,15 +2313,15 @@ class ProjectServiceSpec extends Specification implements ServiceUnitTest<Projec
   <title>blah</title>
   <status>succeed</status>
   <actionType>succeed</actionType>
-  <ctxProject>testproj1</ctxProject>
+  <project>testproj1</project>
```

Comment:
This test data is used in both import and export tests. Changing it here no longer tests the old field names on the import. You should add another XML test string that uses the old field names to ensure they can be imported correctly, such as `<ctxProject>` as well as `<jcJobId>` in the `testLoadReport`

Reply:
But in the `testLoadReport` is being tested `REPORT_XML_TEST1` and `REPORT_XML_TEST1_DEPRECATED`, and the last one holds the old field names `<ctxProject>` and `<jcJobId>`

Reply:
oh ok, perfect